nextcloud_async/api/ocs/shares.py

```diff
@@ -150,14 +150,17 @@ async def create_share(
             # TODO : fill me in
 
         """
-        # try:
-        #     expire_dt = dt.datetime.strptime(expire_date, r'%Y-%m-%d')
-        # except ValueError:
-        #     raise NextCloudException('Invalid date.  Should be YYYY-MM-DD')
-        # else:
-        #     now = dt.datetime.now()
-        #     if expire_dt < now:
-        #         raise NextCloudException('Invalid date.  Should be in the future.')
+
+        # Check exist the expired datetime param for evaluation, avoids this check if expired datatime param not exist
```

Comment:
Let's rewrite to: Checks the expire_date argument exists before evaluation, otherwise continues.